.circleci/config.yml

```diff
@@ -148,6 +175,39 @@ jobs:
           tag: $CIRCLE_TAG
           google-project-id: GOOGLE_PROJECT_ID
 
+      # this could be done by re-tagging the existing image but for ease of migration
+      # I'm duplicating it fully with the intent of removing the above steps entirely
+      # /moz-fx-glam-prod/glam-prod/glam@sha256:705f53c5511c5803c6ea58896a93a97a30dc8defdef19c5d2243c3171a13be25
+      - run:
+          name: Prepare environment variables for OIDC authentication
+          command: |
+            echo 'export GOOGLE_PROJECT_ID="moz-fx-experimenter-prod-6cd5"' >> "$BASH_ENV"
+            echo "export OIDC_WIP_ID=$GCPV2_WORKLOAD_IDENTITY_POOL_ID" >> "$BASH_ENV"
+            echo "export OIDC_WIP_PROVIDER_ID=$GCPV2_CIRCLECI_WORKLOAD_IDENTITY_PROVIDER" >> "$BASH_ENV"
+            echo "export GOOGLE_PROJECT_NUMBER=$GCPV2_WORKLOAD_IDENTITY_POOL_PROJECT_NUMBER" >> "$BASH_ENV"
+            echo "export OIDC_SERVICE_ACCOUNT_EMAIL=$GCP_SERVICE_ACCOUNT_EMAIL" >> "$BASH_ENV"
+
+      - gcp-cli/setup:
+          use_oidc: true
+
+      - gcp-gcr/build-image:
+          workspace-root: dockerfiles/proxy
+          path: dockerfiles/proxy
+          docker-context: dockerfiles/proxy
+          registry-url: us-docker.pkg.dev
+          google-project-id: GLAM_PROJECT_ID
+          repository: glam-prod
+          image: auth0-proxy
+          tag: $CIRCLE_SHA1
+
+      - gcp-gcr/push-image:
```

Comment:
Has this been tested yet? I don't see any images in https://console.cloud.google.com/artifacts/docker/moz-fx-glam-prod/us/glam-prod?project=moz-fx-glam-prod other than the `glam` image.